src/features/advanced/inv-compress-inventory-info.module.css

```
@@ -0,0 +1,14 @@
+.sortControls {
+  padding-top: 0px;
```

```suggestion
  padding-top: 0;
```
Minor improvement: in css, you can amend "px" in "0px"